docs/content/reference/exploiters/powershell.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "PowerShell"
+date: 2021-08-24T12:19:21+03:00
+draft: false
+tags: ["exploit", "windows"]
+---
+### Description
+
+PowerShell Remoting exploit brute forces machines via WinRM service using credentials provided by the user
+(see ["configuration"]({{< ref "/usage/configuration" >}}) for instructions) .
```

Comment:
Can we add some more information here? I don't love just having one sentence in the description where the user probably expects to get a fairly detailed explanation.